modules/mobile/docs/schemas/DependentsRequestDecisions.yml

```diff
@@ -0,0 +1,240 @@
+                - beneficiaryId
+                - birthdayDate
+                - decisionDate
+                - decisionId
+                - dependencyDecisionId
+                - dependencyDecisionType
+                - dependencyDecisionTypeDescription
+                - dependencyStatusType
+                - dependencyStatusTypeDescription
+                - eventDate
+                - firstName
+                - fullName
+                - lastName
+                - modifiedAction
+                - modifiedBy
+                - modifiedDate
+                - modifiedLocation
+                - modifiedProcess
+                - personId
+                - relationshipTypeDescription
+                - sortDate
+                - sortOrderNumber
+                - veteranId
+              properties:
+                awardEffectiveDate:
+                  type: string
+                  nullable: true
+                  example: "2000-06-01T00:00:00.000-05:00"
+                awardEventId:
+                  type: string
+                  nullable: true
+                  example: "60300"
+                awardType:
+                  type: string
+                  example: "CPL"
+                beginAwardEventId:
+                  type: string
+                  nullable: true
+                  example: "25326"
+                beneficiaryId:
+                  type: string
+                  example: "13014883"
+                birthdayDate:
+                  type: string
+                  nullable: true
+                  example: "2000-05-05T00:00:00.000-05:00"
+                decisionDate:
+                  type: string
+                  nullable: true
+                  example: "2006-05-02T09:46:06.000-05:00"
+                decisionId:
+                  type: string
+                  nullable: true
+                  example: "24678"
+                dependencyDecisionId:
+                  type: string
+                  nullable: true
+                  example: "14599"
+                dependencyDecisionType:
+                  type: string
+                  nullable: true
+                  example: "DEPEST"
+                dependencyDecisionTypeDescription:
+                  type: string
+                  nullable: true
+                  example: "Dependency Established"
+                dependencyStatusType:
+                  type: string
+                  nullable: true
+                  example: "SP"
+                dependencyStatusTypeDescription:
+                  type: string
+                  nullable: true
+                  example: "Spouse"
+                eventDate:
+                  type: string
+                  nullable: true
+                  example: "2000-05-05T00:00:00.000-05:00"
+                firstName:
+                  type: string
+                  example: "lauren"
+                fullName:
+                  type: string
+                  nullable: true
```

Comment:
Out of curiosity, how did you decide which ones aren't nullable? In the dependents schema, I didn't specify nullable at all because there was so little information available that I couldn't be sure any attributes would always be present.

Reply:
some of the VCRs available had omitted those fields so there could be more nullable fields.